packages/material/package.json

```diff
@@ -71,32 +71,29 @@
   "peerDependencies": {
     "@jsonforms/core": "^3.0.0-alpha.0",
     "@jsonforms/react": "^3.0.0-alpha.0",
-    "@material-ui/core": "^4.7.0",
-    "@material-ui/icons": "^4.5.1",
-    "@material-ui/pickers": "^3.3.10"
-  },
-  "optionalPeerDependencies": {
-    "@material-ui/lab": "^4.0.0-alpha.56"
+    "@mui/icons-material": "^5.2.0",
+    "@mui/lab": "^5.0.0-alpha.58",
+    "@mui/material": "^5.2.2"
```

Comment:
Do we really depend on the latest versions of all these frameworks? If 5.0.0 is good enough then we should indicate as such to not force users to upgrade from 5.1 to 5.2 for example when we don't actually require them to.